django/contrib/admin/static/admin/js/actions.js

```diff
@@ -143,4 +144,10 @@
         allToggle: "#action-toggle",
         selectedClass: "selected"
     };
+    $(document).ready(function() {
+        if( $("#actions_enabled").length && JSON.parse($("#actions_enabled").text())) {
+            $("tr input.action-select").actions();
```

Comment:
Do we need #actions_enabled? Can we check if `$("tr input.action-select")` isn't null before calling actions() on it?